webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require("path");
+const CleanWebpackPlugin = require("clean-webpack-plugin");
+
+module.exports = {
+  entry: "./src/index.ts",
+  target: "node",
+  output: {
+    path: path.resolve(__dirname, "lib"),
+    filename: "index.js"
+  },
+  devtool: "source-map",
+  resolve: {
+    extensions: [".ts", ".js", ".json"]
+  },
+  optimization: {
+    minimize: true
+  },
+  module: {
+    rules: [
+      {
+        test: /\.ts$/,
+        use: {
+          loader: "babel-loader",
+          options: {
+            presets: ["@babel/preset-typescript"]
```

Comment:
Could we keep all the babel config in here? Rather than in .babelrc and the webpack config? ... Or all in the babelrc. I'm not sure which is better.

Reply:
Moved to one file